lisa/transformers/package_installer.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from dataclasses import dataclass, field
+from pathlib import PurePath
+from typing import Any, Dict, List, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.operating_system import RPMDistro
+from lisa.tools import Rpm
+from lisa.transformers.deployment_transformer import (
+    DeploymentTransformer,
+    DeploymentTransformerSchema,
+)
+from lisa.util import UnsupportedDistroException, field_metadata
+
+
+@dataclass_json()
+@dataclass
+class PackageInstallerSchema(DeploymentTransformerSchema):
+    ...
+
+
+class PackageInstaller(DeploymentTransformer):
+    @classmethod
+    def type_name(cls) -> str:
+        return "package_installer"
+
+    def validate(self) -> None:
```

Comment:
When the validate will be called? It's not in the flow of transformer. It doesn't need here.